### DISCUSSION TITLE
start existing cronjob manually

### DISCUSSION THREAD

**Discussion by mbauhardt** [fabric8io/kubernetes-client#3980] [Q&A] [answered]
hi.
just want to know if the current api offers the possibility to trigger an already existing cronjob manually. the kubectl command is basically `kubectl create job --from=cronjob/<cronjob-name> <job-name>`. 
do you have any hint or code example how i can do it?

thx 
marko

**Comment by rohanKanojia** [comment, marked as answer]
I think this command just creates a job instance with an annotation. Could you please check if this works for you?

https://github.com/rohanKanojia/kubernetes-client-demo/blob/master/src/main/java/io/fabric8/ManualCronJobTrigger.java

**Reply by mbauhardt** [reply]
yes, this example is working after a tiny change. See https://github.com/rohanKanojia/kubernetes-client-demo/pull/8
Not sure if this based on our version `5.12` we are using. anyway thx.

**Reply by rohanKanojia** [reply]
`batch/v1` CronJob resource was introduced in Kubernetes 1.21. Since then `batch/v1beta1` has been deprecated and marked for removal in Kubernetes 1.25 

https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25

**Reply by mbauhardt** [reply]
ok. thx for explaination.
